tests/calculators/test_torsion_calculators.py

```diff
@@ -0,0 +1,47 @@
+from pytest import approx
+import stko
+import stk
+
+
+def test_torsion_matching():
+    """confirm torsions are appropriately mapped from constructed
+    molecules to building blocks
+    """
+
+    def test_polymer(bb1: stk.BuildingBlock, bb2: stk.BuildingBlock):
+        """attach bb1 to bb2 and test torsion mapping"""
+        polymer = stk.ConstructedMolecule(
+            stk.polymer.Linear(
+                building_blocks=(bb1, bb2),
+                repeating_unit="AB",
+                orientations=[0, 0],
```

Comment:
Minor thing, but tuples are better than lists in almost all stk usage - use `(0, 0)`